crates/router/src/configs/settings.rs

```diff
@@ -128,7 +129,12 @@ pub struct CurrencyCountryFilter {
     pub currency: Option<HashSet<api_models::enums::Currency>>,
     #[serde(deserialize_with = "string_set_deser")]
     pub country: Option<HashSet<api_models::enums::CountryCode>>,
-    pub absent_impl: Option<api_models::enums::AbsentImpl>,
+    pub not_available_flows: Option<NotAvailableFlows>,
```

Comment:
The parent struct will have to be renamed, it's not only currency and country filter now.